src/NextGen/DonationForm/DataTransferObjects/DonationFormViewRouteData.php

```diff
@@ -32,7 +34,7 @@ public static function fromRequest(array $request): DonationFormViewRouteData
         $self = new static();
 
         $self->formId = (int)$request['form-id'];
-        $self->formTemplateId = !empty($request['form-template-id']) ? $request['form-template-id'] : '';
+        $self->formSettings = !empty($request['form-settings']) ? $request['form-settings'] : [];
```

Comment:
@kjohnson I was hesitant about adding an array to the query params but I think it's okay. Just remember we are using this both as a post and get request for the form builder and the block rendering so I went ahead and fixed the latter 6161e622

This is what that iframe url will look like going this route:
```
/?givewp-view=donation-form&form-id=2658&form-settings[formTitle]=Test&form-settings[enableDonationGoal]=0&form-settings[enableAutoClose]=0&form-settings[registration]=none&form-settings[goalFormat]=amount-raised&form-settings[templateId]=classic
```

Reply:
Yeah, I had considered encoding the settings, but the flat array works out well with the default array behavior. We can adjust it if it becomes a problem.